healthcheck.go

```diff
@@ -0,0 +1,19 @@
+package dp_redis
+
+import (
+	health "github.com/ONSdigital/dp-healthcheck/healthcheck"
+)
+
+const HealthyMessage = "redis is OK"
+
+func (c *Redis) Checker(state *health.CheckState) error {
+	_, err := c.client.Ping().Result()
+	if err != nil {
+		// Generic error
+		_ = state.Update(health.StatusCritical, err.Error(), 0)
+		return err
+	}
+	// Success
+	_ = state.Update(health.StatusOK, HealthyMessage, 0)
```

Comment:
why ignore the error returned here (and four lines up)?

Reply:
I found this odd too. I was using dp-s3 as a template. In s3 the line was simply: `state.Update(health.StatusOK, HealthyMessage, 0)` but looks like goland plugin for intellij changed it to what you see. Either way, I'm not sure what the purpose of it is.

Reply:
```suggestion
	return state.Update(health.StatusOK, HealthyMessage, 0)
```
is a minimal answer to this, pending further discussion (or maybe a ticket to address this as a bigger issue)